src/sentry/conf/server.py

```diff
@@ -2674,37 +2674,28 @@ def build_cdc_postgres_init_db_volume(settings: Any) -> dict[str, dict[str, str]
             "entrypoint": "/cdc/postgres-entrypoint.sh" if settings.SENTRY_USE_CDC_DEV else None,
         }
     ),
-    "zookeeper": lambda settings, options: (
-        {
-            # On Apple arm64, we upgrade to version 6.x to allow zookeeper to run properly on Apple's arm64
-            # See details https://github.com/confluentinc/kafka-images/issues/80#issuecomment-855511438
-            "image": "ghcr.io/getsentry/image-mirror-confluentinc-cp-zookeeper:6.2.0",
-            "environment": {
-                "ZOOKEEPER_CLIENT_PORT": "2181",
-                "KAFKA_OPTS": "-Dzookeeper.4lw.commands.whitelist=ruok",
-            },
-            "volumes": {"zookeeper_6": {"bind": "/var/lib/zookeeper/data"}},
-            "only_if": "kafka" in settings.SENTRY_EVENTSTREAM or settings.SENTRY_USE_RELAY,
-        }
-    ),
     "kafka": lambda settings, options: (
         {
-            "image": "ghcr.io/getsentry/image-mirror-confluentinc-cp-kafka:6.2.0",
+            "image": "confluentinc/cp-kafka:7.5.0",
```

Comment:
we should be using the image mirrors here

Reply:
https://github.com/getsentry/image-mirror/pull/14